samples/openthread/cli/overlay-logging.conf

```diff
@@ -37,3 +37,7 @@ CONFIG_LOG_BACKEND_RTT=y
 # Disable UART logging backend
 # CONFIG_LOG_BACKEND_UART=n
 CONFIG_SHELL_LOG_BACKEND=n
+
+# mbedTLS logs
```

Comment:
```suggestion
# Enable Mbed TLS logging
```
Also shouldn't we move it a bit higher in the file? maybe before `# Enable OpenThread logging` ?